core/modules/system/tests/system.test

```diff
@@ -129,10 +129,16 @@ class EnableDisableTestCase extends ModuleTestCase {
    * Test that all core modules can be enabled, disabled and uninstalled.
    */
   function testEnableDisable() {
-    // Try to enable, disable and uninstall all core modules, unless they are
-    // hidden or required, or testing-only modules.
+    // Try to enable, disable and uninstall all core modules.
     $modules = system_rebuild_module_data();
     foreach ($modules as $name => $module) {
+      // Check the "version" key matches the Backdrop version to restrict to
+      // core modules only.
+      if ($module->info['version'] !== BACKDROP_VERSION) {
```

Comment:
This will fail if a contrib or custom module happens to use the same version as core. I think that checking the path for `/core` instead may be a better option here. Something like this (untested):

```suggestion
      if (strstr($module->uri, 'core/modules')) {
```

Reply:
Thanks good suggestion! I'll test it and update.